app/controllers/admin/batches_controller.rb

```diff
@@ -12,6 +12,9 @@ def create
     @batch = Batch.new(admin_batch_params)
     if @batch.save
       redirect_to admin_roast_url(@batch.roast), notice: 'Batch was successfully created.'
+      if !(Rails.env.test?)
```

Comment:
We should avoid doing that.
That means that in the controller test you need to stub that method instead.

Reply:
It can overload tests to stub in each single test but maybe a pattern will rise and a refactor will be possible. If not then you need to stub the method on all the tests that call  the `create` batch controller method